package.json

```
@@ -15,10 +15,8 @@
   "scripts": {
     "eslint:fix": "eslint . --fix",
     "eslint:check": "eslint . --cache",
```

We should consider here to change the way of the linting. This checks only the new code part.
It would be good to introduce a new script, that checks the full code base. This could be used at every further lint changes.
```suggestion
    "eslint:full-check": "eslint ./auth/* ./lib/* ./cloudant/*",
```